packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/FIAPRequestHandler.h

```diff
@@ -2,9 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#import <FLTRequestHandlerProtocol.h>
 #import <Foundation/Foundation.h>
 #import <StoreKit/StoreKit.h>
+#import "FLTRequestHandlerProtocol.h"
```

Comment:
@LouiseHsu All of these headers are public due to this (I know this predates your migration):
https://github.com/flutter/packages/blob/a87eddf0c281d5d017093125c1a8d9161257741f/packages/in_app_purchase/in_app_purchase_storekit/darwin/in_app_purchase_storekit.podspec#L21

https://guides.cocoapods.org/syntax/podspec.html#public_header_files

Public headers should not use file (quote) imports, but should use angle brackets.
https://developer.apple.com/documentation/xcode/build-settings-reference#Quoted-Include-In-Framework-Header

Also the [import order](https://google.github.io/styleguide/objcguide.html#order-of-includes) should be
```
#import <Foundation/Foundation.h>
#import <StoreKit/StoreKit.h>
```
then
```suggestion
#import <in_app_purchase_storekit.framework/FLTRequestHandlerProtocol.h>
```

When in doubt, open up an Apple header and copy their pattern.  For example, this is StoreKit's `SKAdImpression` header:
```objc
//
//  SKAdImpression.h
//  StoreKit
//
//  Copyright © 2020 Apple Inc. All rights reserved.
//

#import <Foundation/Foundation.h>
#import <StoreKit/StoreKitDefines.h>
```

1. Can you think through [which headers should actually be public](https://google.github.io/styleguide/objcguide.html#keep-the-public-api-simple), and only expose that from the podspec (and later from the Package.swift file)?
1. Audit the headers to make them angle bracket imports.
2. If it's done in this PR, make sure the CHANGELOG and pubspec version is also update.

Reply:
Ah. I missed this comment. Ill make an issue to fix all the public headers.